nims-notes-things/src/Form.js

```diff
@@ -22,57 +24,81 @@ const Form = (props) => {
     };
     setFormData(newForm);
   };
+  const handleOnSubmit = (event) => {
+    validateSubmit(event);
+  };
+
+  const validateSubmit = (event) => {
```

Comment:
See my comment here for an approach to this issue: https://github.com/alexholmes99/notepadchallenge/pull/6#discussion_r813051361